resources/views/database.blade.php

```diff
@@ -2,7 +2,52 @@
 <html>
     <head>
         <title>Monty Hall - Database Page</title>
+        <link href="{{ URL::asset('css/database.css') }}" rel="stylesheet"/>
     </head>
-    <x-navigation/>
-    <h1>Monty Hall - Database Page</h1>
+    <body>
+        <x-navigation/>
+        <h1 class="pagetitle">Monty Hall - Database Page</h1>
```

Comment:
I think this header should be changed to say "Results Database" instead of the generic "Monty Hall - Database Page"

Reply:
yeah I had left it cuz I wasn't sure if we wanted to make it the same as my mockup or not